apps/allocations/app/components/Card/Budget.js

```diff
@@ -0,0 +1,213 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { BigNumber } from 'bignumber.js'
+
+import {
+  Card,
+  ContextMenu,
+  ContextMenuItem,
+  IconEdit,
+  IconPlus,
+  IconProhibited,
+  IconView,
+  ProgressBar,
+  Text,
+  useTheme,
+} from '@aragon/ui'
+import { displayCurrency } from '../../utils/helpers'
+
+import { BASE_CARD_WIDTH, CARD_STRETCH_BREAKPOINT } from '../../utils/responsive'
+
+const Budget = ({
+  id,
+  name,
+  amount,
+  currency,
+  allocated,
+  inactive,
+  onNewAllocation,
+  onEdit,
+  onDeactivate,
+  onReactivate,
+  screenSize,
+}) => {
+  const theme = useTheme()
+  const newAllocation = () => {
+    onNewAllocation(id, name, amount)
+  }
+  const edit = () => {
+    onEdit(id)
+  }
+  const deactivate = () => {
+    onDeactivate(id)
+  }
+
+  const reactivate = () => {
+    onReactivate(id)
+  }
+
+  if (inactive) {
+    return (
+      <StyledCard screenSize={screenSize}>
+        <MenuContainer>
+          <ContextMenu>
+            <ContextMenuItem onClick={reactivate}>
+              <IconView />
+              <ActionLabel>Reactivate</ActionLabel>
+            </ContextMenuItem>
+          </ContextMenu>
+        </MenuContainer>
+        <CardTitle theme={theme}>{name}</CardTitle>
+        <StatsContainer>
+          <StyledStats>
+            <StatsValueBig theme={theme}>
+              <Text>Inactive</Text>
+            </StatsValueBig>
+          </StyledStats>
+        </StatsContainer>
+      </StyledCard>
```

Comment:
This makes the developer so much easier to read and maintain the component instead of using a bunch of `div`s with `css` props spreading over several lines and lots of logic or conditions inside, please stick to this approach instead of what was done in the other components. ❤️